server/src/main/java/org/elasticsearch/common/hash/Murmur3Hasher.java

```diff
@@ -39,34 +36,41 @@ public Murmur3Hasher(long seed) {
      * {@link #digest()} method should be called to complete the hash calculation.
```

Comment:
Should this comment say `digestHash` instead of `digest`?

Reply:
Updated in c1a2430.